java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java

```diff
@@ -355,6 +355,11 @@ public class ConfigDefaults {
      */
     public static final String DEFAULT_DOCS_LOCALE = "web.docs_locale";
 
+    /**
+     * Specify if custom channels are synced together with vendor channels
+     */
+    public static final String UNIFY_CUSTOM_CHANNEL_MANAGEMENT = "java.unify_custom_channel_management";
```

Comment:
You can also specify the default in the config-defaults file: java/conf/rhn_java.conf

Add the key there with true or false and add a description what this is doing.